tests/test_auth.py

```diff
@@ -1,13 +1,18 @@
 import flask
-from flask_resty import (
-    Api, ApiError, AuthenticationBase, GenericModelView,
-    HasAnyCredentialsAuthorization, HasCredentialsAuthorizationBase,
-)
-from flask_resty.testing import assert_response
 from marshmallow import fields, Schema
 import pytest
 from sqlalchemy import Column, Integer, String
 
+from flask_resty import (
```

Comment:
the nice thing there is, when things break onto multiple lines, if you add something in the middle, you may need to re-wrap the line, which clutters up the diff